.github/dependabot.yml

```diff
@@ -1,22 +0,0 @@
-version: 2
-updates:
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "gomod"
-    directory: "/"
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 2
-    ignore:
-      # OpenTelemetry needs special handling due to a temporary fork; exempt
-      # from Dependabot auto-updating.
-      - dependency-name: "github.com/open-telemetry/opentelemetry-collector-contrib/*"
-      - dependency-name: "go.opentelemetry.io/collector"
```

Comment:
OTel dependencies still likely to be ones we want to ignore with renovate.

Reply:
Opened #3185 to track this for when we enable renovate for go deps!